### PR TITLE
Skip things that have been done before and skip folders

### LINKED ISSUE #232: Issues during the corrosion data import
After executing the "python -m pasta_eln.serverActions" command and following all steps, a following error occurs:

[1] couchDB_backup_127001_20240316T140124.zip
Which file to use for restored? (1-1) 1
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/htsybenko/Temp/envir/lib/python3.10/site-packages/pasta_eln/serverActions.py", line 582, in <module>
    main()
  File "/home/htsybenko/Temp/envir/lib/python3.10/site-packages/pasta_eln/serverActions.py", line 559, in main
    restoreCouchDB()
  File "/home/htsybenko/Temp/envir/lib/python3.10/site-packages/pasta_eln/serverActions.py", line 456, in restoreCouchDB
    docID = fileParts[1]
IndexError: list index out of range

### PR DESCRIPTION
**Pull request by SteffenBrinckmann**
Fix #232

### CODE REVIEW

File: pasta_eln/serverActions.py [hunk 1]
```diff
@@ -486,7 +486,7 @@ def restoreCouchDB(location:str='', userName:str='', password:str='', fileName:s
     #second run through: create attachments
     for fileI in files:
       fileParts = fileI.split('/')[1:]
-      if fileParts==['pastaELN.json']: #do not recreate file, it is only there for manual recovery
+      if fileParts==['pastaELN.json'] or fileParts[-1]=='' or fileParts[-2]=='_design': #do not recreate file, it is only there for manual recovery
```
Comment (jmurugan-fzj) [inline pasta_eln/serverActions.py:489]: When is fileParts[-1]=='' happening? This means the last part is an empty string, what kind of file is it?

Comment (SteffenBrinckmann) [reply pasta_eln/serverActions.py:489]: Those are folders, as these end with a '/' and should be skipped

Comment (SteffenBrinckmann) [reply pasta_eln/serverActions.py:489]: Hanna saw, that these exist in her files and caused an warning, which should not be raised.

**Review by jmurugan-fzj — APPROVED** [review]

File: pasta_eln/serverActions.py (hunk 1 above)
Comment (SteffenBrinckmann) [reply pasta_eln/serverActions.py:489]: @jmurugan-fzj :+1: I corrected the code accordingly and files only holds files.

**Pull request merged**